### ISSUE TITLE
Unsupported browser message is cached after using Chrome dev tools ipad emulation

### ISSUE THREAD

**Issue by Artur-** [vaadin/flow#10282]
### Description of the bug / feature
If you open Chrome dev tools, set it to emulate an ipad and load any Vaadin 19 app, you will see "I'm sorry but your browser is not supported"

The user agent is
```
"Mozilla/5.0 (iPad; CPU OS 11_0 like Mac OS X) AppleWebKit/604.1.34 (KHTML, like Gecko) Version/11.0 Mobile/15A5341f Safari/604.1"
```

### Versions:
    - Vaadin / Flow version: 19

**Comment by Artur-** [comment]
If you go back to normal Chrome mode, you still see the unsupported browser page

**Comment by caalador** [comment]
The version requirement for safari after #6856 has been updated to 13+ and the version in the given string is 11.0 and even the os is set as 11_0 where as most ipads should be on 13 or 14 now.
Even Gmail sees the ipad test version as a legacy browser.

I don't see what we could do for this as it's clearly a chrome issue of reporting an old version as the actual Ipad browser version is at 14.0

**Comment by Artur-** [comment]
The bigger issue is that the ”unsupported browser page” is cached in a service worker(?) and is shown also after disabling ipad mode and refreshing

**Comment by caalador** [comment]
On windows at least this required to open the app for the first time in ipad mode after which it got stuck in the unsupported mode and only clearing site data for localhost made the application work for refreshing.

**Comment by bsutton** [comment]
I'm having the same issue thought I would post my work around.

Chrome allows you to modify the user agent.

This article explains how to change your agent:
https://umaar.com/dev-tips/234-custom-user-agent/

I set my to be chrome 100.

Mozilla/5.0 (Linux; U; Android 8.1.0; en-US; Nexus 6P Build/OPM7.181205.001) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/100.0.2987.108 UCBrowser/12.11.1.1197 Mobile Safari/537.36

I selected an existing user agent from the drop list and just updated the version no.
